inc/saifdb.h

```diff
@@ -229,11 +229,11 @@ typedef enum _sai_fdb_flush_entry_type_t
  *
  * For example:
  *
- * 1) Flush all entries in FDB table - Do not specify any attribute
- * 2) Flush all entries by bridge port - Set #SAI_FDB_FLUSH_ATTR_BRIDGE_PORT_ID
- * 3) Flush all entries by VLAN - Set #SAI_FDB_FLUSH_ATTR_BV_ID with object id as vlan object
- * 3) Flush all entries by bridge - Set #SAI_FDB_FLUSH_ATTR_BV_ID with object id as bridge object
- * 4) Flush all entries by bridge port and VLAN - Set #SAI_FDB_FLUSH_ATTR_BRIDGE_PORT_ID
+ * 1) Flush all dynamic entries in FDB table - Do not specify any attribute
+ * 2) Flush dynamic entries by bridge port - Set #SAI_FDB_FLUSH_ATTR_BRIDGE_PORT_ID
+ * 3) Flush dynamic entries by VLAN - Set #SAI_FDB_FLUSH_ATTR_BV_ID with object id as vlan object
+ * 3) Flush dynamic entries by bridge - Set #SAI_FDB_FLUSH_ATTR_BV_ID with object id as bridge object
+ * 4) Flush dynamic entries by bridge port and VLAN - Set #SAI_FDB_FLUSH_ATTR_BRIDGE_PORT_ID
  *    and #SAI_FDB_FLUSH_ATTR_BV_ID
  * 5) Flush all static entries by bridge port and VLAN - Set #SAI_FDB_FLUSH_ATTR_ENTRY_TYPE,
  *    #SAI_FDB_FLUSH_ATTR_BRIDGE_PORT_ID, and #SAI_FDB_FLUSH_ATTR_BV_ID
```

Comment:
i think this also don't specifies all scenarios since if only SAI_FDB_FLUSH_ATTR_BRIDGE_PORT_ID is set, then the dynamic entries will be flushed, butwhen we set SAI_FDB_FLUSH_ATTR_BRIDGE_PORT_ID and SAI_FDB_FLUSH_ATTR_ENTRY_TYPE set to static then static entries will be flused or all entries if type will be all so those descriptions it is in assumption that type is not dpecified

so meybe it should be something similar to this:

By default (when SAI_FDB_FLUSH_ATTR_ENTRY_TYPE is not specified) only dynamic entries are affected when doinf flush. By setting SAI_FDB_FLUSH_ATTR_ENTRY_TYPE only dynamic entries can be flushed, static or both, depending on attribute value. Futher more, setting other attributes, can restrict flush only to speciffic enrtires, filetered by bridge port, vlan or bridge. Combinded with type atribute, this can affect static,dynamic or both entries. If bride_port_id attribute is not set, it's not participating into filter. Same for bv_id attribute.